docs/connect/odbc/linux-mac/data-access-tracing-with-the-odbc-driver-on-linux.md

```diff
@@ -16,34 +16,35 @@ ms.author: genemi
 manager: craigg
 ---
 # Data Access Tracing with the ODBC Driver on Linux and macOS
+
 [!INCLUDE[Driver_ODBC_Download](../../../includes/driver_odbc_download.md)]
 
 The unixODBC Driver Manager on macOS and Linux supports tracing of ODBC API call entry and exit of the ODBC Driver for [!INCLUDE[ssNoVersion](../../../includes/ssnoversion-md.md)].
 
 To trace your application's ODBC behavior, edit the `odbcinst.ini` file's `[ODBC]` section to set the values `Trace=Yes` and `TraceFile`
 to the path of the file which is to contain the trace output; for example:
 
-```  
+```ini
 Trace=Yes
 TraceFile=/home/myappuser/odbctrace.log
-```  
+```
 
 (You may also use `/dev/stdout` or any other device name to send trace output there instead of to a persistent file.) With the above settings, every time an application loads the unixODBC Driver Manager, it will record all the ODBC API calls which it performed into the output file.
 
 After you finish tracing your application, remove `Trace=Yes` from the `odbcinst.ini` file to avoid the performance penalty of tracing, and ensure any unnecessary trace files are removed.
-  
-Tracing applies to all applications that use the driver in `odbcinst.ini`. To not trace all applications (for example, to avoid disclosing sensitive per-user information), you can trace an individual application instance by providing it the location of a private `odbcinst.ini`, using the `ODBCSYSINI` environment variable. For example:  
-  
-```  
-$ ODBCSYSINI=/home/myappuser myapp
-```  
-  
+
+Tracing applies to all applications that use the driver in `odbcinst.ini`. To not trace all applications (for example, to avoid disclosing sensitive per-user information), you can trace an individual application instance by providing it the location of a private `odbcinst.ini`, using the `ODBCSYSINI` environment variable. For example:
+
+```bash
+export ODBCSYSINI=/home/myappuser myapp

```

Comment:
changed based on the paragraph describing that this is supposed to be set as an environment variable. The value might need to be quoted though

Reply:
Actually, this is the inline invokation for the app with the variable set, so i'll undo this